lib/db.js

```diff
@@ -449,9 +449,9 @@ Store.prototype.update = function (query, object, fn) {
   debug('update - command', command);
 
   collection(this, function (err, col) {
-      col.update(query, command, {multi: multi}, function(err) {
+    col.update(query, command, { multi: multi }, function (err, r) {
       store.identify(query);
-      fn(err);
+      fn(err, r ? { n: r.result.n } : null);
```

Comment:
What would you suggest? I picked `n` because it made sense for both updates and deletions and wouldn't leak too much info if it got out in the body as a HTTP response (for a DELETE or logout).